<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionFont"/>
    <addaction name="actionItalique"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionBold"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionItalique"/>
   <addaction name="separator"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/new.png</normaloff>:/Images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/open.png</normaloff>:/Images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/save.png</normaloff>:/Images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/save_as.png</normaloff>:/Images/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/exit.png</normaloff>:/Images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/pencil.png</normaloff>:/Images/images/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/font.png</normaloff>:/Images/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/copy.png</normaloff>:/Images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/cut.png</normaloff>:/Images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../Images/images/paste.png</normaloff>../Images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/edit_undo.png</normaloff>:/Images/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/edit_redo.png</normaloff>:/Images/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionItalique">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/italic.png</normaloff>:/Images/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italique</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/underline.png</normaloff>:/Images/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/bold.png</normaloff>:/Images/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionWeight">
   <property name="text">
    <string>Weight</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/images/info.png</normaloff>:/Images/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
